From aa4da77a55c423bd984f9188f43fe16e1d12ae68 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:01:37 -0400
Subject: [usb] host: Use existing macros instead of hard-coded values in uhci-debug.c

Message-id: <1410887463-20674-26-git-send-email-dzickus@redhat.com>
Patchwork-id: 95659
O-Subject: [RHEL7 PATCH 025/591] USB/host: Use existing macros instead of hard-coded values in uhci-debug.c
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit f1125f81feca956288ef4ae4fcf47961865ab2e2
Author: Deng-Cheng Zhu <dengcheng.zhu@imgtec.com>
Date:   Fri Oct 4 15:14:34 2013 -0700

    USB/host: Use existing macros instead of hard-coded values in uhci-debug.c

    Now that UHCI IO registers have been defined in uhci-hcd.h, use them.

    Reviewed-by: James Hogan <james.hogan@imgtec.com>
    Signed-off-by: Deng-Cheng Zhu <dengcheng.zhu@imgtec.com>
    Acked-by: Alan Stern <stern@rowland.harvard.edu>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/uhci-debug.c b/drivers/usb/host/uhci-debug.c
index 4557375..8e239cd 100644
--- a/drivers/usb/host/uhci-debug.c
+++ b/drivers/usb/host/uhci-debug.c
@@ -310,14 +310,14 @@ static int uhci_show_status(struct uhci_hcd *uhci, char *buf, int len)
  unsigned short portsc1, portsc2;
 
 
- usbcmd    = uhci_readw(uhci, 0);
- usbstat   = uhci_readw(uhci, 2);
- usbint    = uhci_readw(uhci, 4);
- usbfrnum  = uhci_readw(uhci, 6);
- flbaseadd = uhci_readl(uhci, 8);
- sof       = uhci_readb(uhci, 12);
- portsc1   = uhci_readw(uhci, 16);
- portsc2   = uhci_readw(uhci, 18);
+ usbcmd    = uhci_readw(uhci, USBCMD);
+ usbstat   = uhci_readw(uhci, USBSTS);
+ usbint    = uhci_readw(uhci, USBINTR);
+ usbfrnum  = uhci_readw(uhci, USBFRNUM);
+ flbaseadd = uhci_readl(uhci, USBFLBASEADD);
+ sof       = uhci_readb(uhci, USBSOF);
+ portsc1   = uhci_readw(uhci, USBPORTSC1);
+ portsc2   = uhci_readw(uhci, USBPORTSC2);
 
  out += sprintf(out, "  usbcmd    =     %04x   %s%s%s%s%s%s%s%s\n",
   usbcmd,
-- 
1.7.1